metadata-app/src/api/index.js

```diff
@@ -9,6 +9,21 @@ const licenses = require('../components/RequiredMetadata/licenses.json');
  */
 const clone = (param) => JSON.parse(JSON.stringify(param));
 
+/**
+ * Converts mm/dd/yyyy -> yyyy-mm-dd
+ */
+const convertFwdSlashIntoDashDateFormat = (dateStr) => {
+  let converted = dateStr;
+  const dateParts = dateStr.split('/');
```

Comment:
Not sure about this approach, maybe we can parse dates using `Date` class of JS?

Reply:
The previous implementation was based on that and we used to get wrong date output to ckan. The bug was depending on the users timezone the output might be different, for example in my case if I create:
`new Date('01/13/2020')`
It outputs:  
`2020-01-12T20:00:00.000Z`

So you see the day is 12 instead of **13**. So that's why I have come up with this implementation, maybe it's not the ideal option, but basically we are dealing with the way we format the date.

Reply:
@kmanaseryan ok, I got it, didn't look at previous implementation. Yes, that's quite common challenge when dealing with timezones etc. I believe that the submitted date format is persistent so this implementation should work fine. The only thing I'd make sure that we have tests for that.